tools/nuget/ebpf-for-windows.nuspec.in

```diff
@@ -28,6 +28,7 @@
 		<file src="ebpfapi.dll" target="build\native\bin"/>
 		<file src="ebpfapi.pdb" target="build\native\bin"/>
 		<file src="ebpfapi.lib" target="build\native\lib"/>
+		<file src="ebpf_store_helper_km.lib" target="build\native\lib"/>
```

Comment:
Why does this lib need to be exposed to other projects?  Shouldn't it just be internal?

Reply:
This lib exposes KM store APIs which need to be used by extensions to populate the store with the program / attach type they are implementing. netebpfext uses these APIs today. xdp-for-windows, for instance, should also use store APIs to populate the store.

Reply:
Thanks for the explanation.   I am now convinced that the PR description is wrong when it says:
> **Documentation**
> n.a.

In particular https://github.com/microsoft/ebpf-for-windows/blob/main/docs/eBpfExtensions.md#27-registering-program-types-and-attach-types needs to be updated in this PR.